apps/super-app/src/App.test.tsx

```diff
@@ -3,27 +3,30 @@ import { render, screen } from '@testing-library/react';
 import App from './App';
 import { MemoryRouter } from 'react-router-dom';
 
-test('renders a nonsense homepage', () => {
+test('renders LandingPage component on load homepage', () => {
```

Comment:
> `renders LandingPage component on load homepage`

This doesn't read right. For consistency with the other test maybe `renders LandingPage component on /`?
Also, the test isn't actually asserting that the component is being rendered, but that the expected elements are on the page, so maybe something like `renders support cards on /`?